### PR TITLE
Add/improve script docstrings.

### PR DESCRIPTION
**Pull request by vext01**
Do not merge yet.

Sarah, I've reviewed all of the scripts in the `bin/` directory of this repo. I've added one docstring and improved many.

There are still some scripts which I was not involved with which are missing docstrings. If you could add docstrings for these (in this branch), we can squash and merge later.

Thanks

### CODE REVIEW

**Comment by snim2** [comment]
OK, so these are scripts I don't know much about:

  * `check_amperfs`
  * `find_v8_git_versions`
  * `fix_v8_versions`

The `plot_krun_results` file is already commented, so I think that's "my" scripts done.

**Comment by ltratt** [comment]
Those 3 scripts look like they're the result of Edd's fair hands. @vext01 can you add comments to them?

**Comment by vext01** [comment]
Those are indeed mine. The former two have docstrings:

https://github.com/softdevteam/warmup_experiment/blob/1f1d9c6277ee9580aab2062875edfdcf0c7d98d0/bin/check_amperfs#L44-L48

https://github.com/softdevteam/warmup_experiment/blob/1f1d9c6277ee9580aab2062875edfdcf0c7d98d0/bin/find_v8_git_versions#L41-L48

The last one is generated, but says which script made it:

https://github.com/softdevteam/warmup_experiment/blob/1f1d9c6277ee9580aab2062875edfdcf0c7d98d0/bin/fix_v8_versions#L1

Good enough for me.

**Comment by vext01** [comment]
By the way, we have merge conflicts on a couple of Sarah's scripts.

**Comment by snim2** [comment]
Yes, just because we merged a couple of PRs after this one was raised. So, maybe I should squash and rebase this one?

**Comment by vext01** [comment]
Please.

**Comment by ltratt** [comment]
Is this ready for review? If so, we should rebase (and, if you guys want to, squash) before review I think.

**Comment by snim2** [comment]
Yes, I'll rebase since the conflicting files are ones I was working with. Squashed commits coming up...

**Comment by snim2** [comment]
Squashed and rebased...

**Comment by ltratt** [comment]
OK, so this looks good to me. Merge as-is?

**Comment by snim2** [comment]
If @vext01 is happy then yes please

File: bin/plot_truncated_iterations [hunk 1]
```diff
@@ -37,6 +37,11 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+"""
+Plot a chart describing the effect of re-running an experiment with fewer pexecs.
```
Comment (vext01) [inline bin/plot_truncated_iterations]: pexecs -> iterations?

**Comment by vext01** [comment]
One comment, I think a typo.

**Comment by snim2** [comment]
Thanks, fixed! OK to squash again?

**Comment by ltratt** [comment]
Yep, please squash and then I'll merge.

**Comment by snim2** [comment]
Squashed.

**Review by ltratt — APPROVED** [review]

**Pull request merged**